config/routes.rb

```diff
@@ -154,6 +154,7 @@
     post "/jenkins/:token" => "jenkins#create", as: :jenkins_deploy
     post "/buildkite/:token" => "buildkite#create", as: :buildkite_deploy
     post "/github/:token" => "github#create", as: :github_deploy
+    post "/ci_pipeline/:token" => "ci_pipeline#create", as: :ci_pipeline_deploy
```

Comment:
ci_pipeline seems very specific ... I'd have expected something like `generic` ... whatever calls this does not need to be part of a ci-pipeline

Reply:
👍